.changelog/556.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+Upgrade Go to version 1.19
```

Comment:
Since this PR does a fair bit more that just upgrade Go, should we include those details in the changelog?

Reply:
The rest of the changes are CI changes

Reply:
The rest of the changes are CI changes that shouldn't affect consumers of the provider. We could include them in the changelog, but I think they'd be more noise than useful.